edb/pgsql/compiler/relctx.py

```diff
@@ -183,7 +182,7 @@ def include_rvar(
         overwrite_path_rvar: bool=False,
         pull_namespace: bool=True,
         flavor: str='normal',
-        aspects: Optional[Tuple[str, ...]]=None,
+        aspects: Optional[Collection[str]]=None,
```

Comment:
Alas, `Collection[str]` includes `str` itself, and I'm always uneasy when there's a signature like this.  Perhaps `Tuple[str, ...] | AbstractSet[str] | None`?